pkg/storaged/fsys-panel.jsx

```diff
@@ -169,6 +169,7 @@ export class FilesystemsPanel extends React.Component {
             <OptionalPanel id="mounts" className="storage-mounts"
                 title={_("Filesystems")}>
                 <ListingTable
+                    gridBreakPoint="grid-xl"
```

Comment:
Should this be a different breakpoint?

I still see unoptimal layouts like this:

![image](https://user-images.githubusercontent.com/10246/234904744-937dac0b-e87e-42d9-8175-88027f532d7a.png)

It's definitely better with this PR (no overlaps), though!

Reply:
Oh, whoops. This isn't the right place (I've been playing with different values.) We'd want to change it elsewhere, I think.